pkg/controllers/cloud/networkpolicy_sync.go

```diff
@@ -28,59 +28,64 @@ import (
 
 // sync synchronizes securityGroup memberships with cloud.
 // Return true if cloud and controller has same membership.
-func (s *securityGroupImpl) syncImpl(csg cloudSecurityGroup, c *securitygroup.SynchronizationContent, membershipOnly bool,
+func (sg *securityGroupImpl) syncImpl(csg cloudSecurityGroup, syncContent *securitygroup.SynchronizationContent, membershipOnly bool,
 	r *NetworkPolicyReconciler) bool {
 	log := r.Log.WithName("CloudSync")
-	if c != nil {
-		s.state = securityGroupStateCreated
-		cMembers := make([]*securitygroup.CloudResource, 0, len(c.Members))
-		for i := range c.Members {
-			cMembers = append(cMembers, &c.Members[i])
+	if syncContent != nil {
+		sg.state = securityGroupStateCreated
+		syncMembers := make([]*securitygroup.CloudResource, 0, len(syncContent.Members))
+		for i := range syncContent.Members {
+			syncMembers = append(syncMembers, &syncContent.Members[i])
 		}
 
-		nMembers := s.members
-		if len(cMembers) > 0 && cMembers[0].Type == securitygroup.CloudResourceTypeNIC {
-			nMembers, _ = r.getNICsOfCloudResources(s.members)
+		cachedMembers := sg.members
+		if len(syncMembers) > 0 && syncMembers[0].Type == securitygroup.CloudResourceTypeNIC {
+			cachedMembers, _ = r.getNICsOfCloudResources(sg.members)
 		}
-		if compareCloudResources(nMembers, cMembers) {
-			log.V(1).Info("Same SecurityGroup found", "Name", s.id.Name, "State", s.state)
+		if compareCloudResources(cachedMembers, syncMembers) {
+			log.V(1).Info("Everything in sync with cloud", "Name", sg.id.Name, "State", sg.state)
```

Comment:
actually I want logging to be reduced.. This will again show that member is synced.. At every interval the minimal logging I am looking for is
1) Syncing cloud entities.. 
2) Figure out this entity requires re-sync and just dump that.

With that, I would implicitly know those entities are not in sync, rather than seeing success logs for all.

Reply:
Removed the logging when members are in sync. Otherwise, added a log.